.github/workflows/jekyll.yaml

```diff
@@ -27,7 +27,7 @@ on:
 
 jobs:
   jekyll:
-    timeout-minutes: 10
+    timeout-minutes: 15
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
```

Comment:
I wonder if some of this would be faster if we updated to use v3 for the actions. Could make that change as well (or use whatever is the latest, if it's newer than v3).